src/uct/api/v2/uct_v2.h

```diff
@@ -228,6 +280,24 @@ uct_iface_estimate_perf(uct_iface_h tl_iface, uct_perf_attr_t *perf_attr);
 ucs_status_t uct_md_mem_dereg_v2(uct_md_h md,
                                  const uct_md_mem_dereg_params_t *params);
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Check if remote iface address is reachable.
+ *
+ * This function checks if a remote address can be reached from a local interface.
+ * If the function returns true, it does not necessarily mean a connection and/or
+ * data transfer would succeed, since the reachability check is a local operation
+ * it does not detect issues such as network mis-configuration or lack of connectivity.
+ *
+ * @param [in]  iface       Local interface to check reachability from.
+ * @param [in]  params      Operation parameters, see @ref
+ *                          uct_iface_is_reachable_params_t.
+ *
+ * @return Nonzero if reachable, 0 if not.
+ */
+int uct_iface_is_reachable_v2(const uct_iface_h iface,
```

Comment:
iface_h does not need to be const (we didn't do it so far in the API)

Reply:
fixed